CMakeLists.txt

```diff
@@ -151,6 +155,9 @@ enable_clang_tidy(
     EXTRA_ARGS
 
 )
+
```

Comment:
can you remove ClangTidy in the above lines as well? we shouldn't need to include it.

Reply:
Removed